site/content/docs/docker/webswing.md

```diff
@@ -26,6 +26,38 @@ The ZAP UI acts in just the same way as it does when running 'natively'.
 
 The performance is surprisingly good, although a little bit slower. You can still move and resize the windows and some (but not all) of the keyboard shortcuts will work.
 
+## Proxying through ZAP
+
 You can even proxy your browser through ZAP, via the URL [http://localhost:8090](http://localhost:8090)
 
+However it is not possible to launch browsers via ZAP started via Webswing so in order to proxy https based sites you need to start the docker container with a local drive mapped to `/zap/wrk` e.g. using:
+
+*  `docker run -v $(pwd):/zap/wrk/:rw -u zap -p 8080:8080 -p 8090:8090 -i owasp/zap2docker-stable zap-webswing.sh`
+
+When you do this ZAP will create 2 files on your mapped drive:
+
+* owasp_zap_root_ca.crt - the public ZAP Root CA certificate
+* owasp_zap_root_ca.key - the private ZAP Root CA certificate
+
+You will then need to configure one of your browsers to [proxy via ZAP](/docs/desktop/start/proxies/) 
+and [import the public ZAP Root CA certificate](/docs/desktop/ui/dialogs/options/dynsslcert/#install) so that it is trusted to sign websites.
+
+It is recommended that you keep a separate browser profile for proxying through ZAP.
+
+If you restart the docker container using the same mapped drive then ZAP will automatically import the private ZAP Root CA certificate 
+so you will not need to import it into your browser again.
+
+
 Note that you will need to visit http://localhost:8080/zap _before_ you can use ZAP as a proxy - ZAP will only start to run when you visit that URL.
+
+## ZAP Command Line Options
+
+When you run ZAP using Webswing then the following ZAP Command Line Options will be used:
+
```

Comment:
I think you missed pasting a line here?

Reply:
Er, no, its supposed to refer to the following lines :/

Reply:
Oh I guess I was anticipating the "..the following default options are used...." line.... okay no problem